pyhealth/medcode/pretrained_embeddings/kg_emb/examples/train_kge_model.py

```diff
@@ -0,0 +1,142 @@
+from pyhealth.medcode.pretrained_embeddings.kg_emb.datasets import UMLSDataset, split
```

Comment:
Please move all the examples under the example/ folder. You should not create a new eco-system under the kg_emb folder.......